packages/react-components/react-tabster/src/focus/createCustomFocusIndicatorStyle.ts

```diff
@@ -24,19 +24,9 @@ export interface CreateCustomFocusIndicatorStyleOptions {
  */
 export function createCustomFocusIndicatorStyle<TStyle extends GriffelStyle | GriffelResetStyle>(
   style: TStyle,
-  {
-    selector = defaultOptions.selector,
-    enableOutline = false,
-  }: CreateCustomFocusIndicatorStyleOptions = defaultOptions,
+  { selector = defaultOptions.selector }: CreateCustomFocusIndicatorStyleOptions = defaultOptions,
```

Comment:
@ling1726 / @layershifter is it OK to remove `enableOutline` here entirely, or would you prefer to keep it and mark it deprecated?

Reply:
we should deprecate `enableOutline` (totally my fault for not thinking it through completely when it was introduced)  since the utility is a part of the public API .🙃